extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/util/MultiStageQueryContext.java

```diff
@@ -197,10 +196,7 @@ public static int getMaxNumTasks(final QueryContext queryContext)
 
   public static Object getDestination(final QueryContext queryContext)
   {
-    return queryContext.get(
-        CTX_DESTINATION,
-        DEFAULT_DESTINATION
-    );
+    return queryContext.get(CTX_DESTINATION);
```

Comment:
Was this intentional ?

Reply:
Yes, this is a minor refactor, the default destination is null, however, we needn't supply null to the default value in queryContext.get, as that's what queryContext.get() returns if it doesn't find the context param that we have requested.